libr/asm/arch/avr/format.c

```diff
@@ -27,7 +27,9 @@
 #include <string.h>
 #include <stdlib.h>
 #include "format.h"
-
+#ifdef _MSC_VER
+#define strcasecmp stricmp
```

Comment:
errr. this strcasecmp thing should be used with a define in r_types.h like this

#if _MSC_VER
#define R_STRCASECMP stricmp
#else 
#define R_STRCASECMP strcasecmp
#endif 

or maybe we can just define static inline functions in r_util.h